code/src/Components/Album.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react';
+import { Artists } from './Artists.js';
+
+export const Album = (props) => {
+    console.log(props);
```

Comment:
remember to remove console.logs  from your code after you finish debugging your code.